src/typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft.azure/openapi-validator",
-    "version": "1.0.5",
+    "version": "1.2.0",
```

Comment:
Please work with Jianye to change the version together and why use 1.2.0 directly?

Reply:
Ok. We will create a separate PR to bump version and update changelog.